camille/source/windiris.py

```diff
@@ -4,12 +4,26 @@
 from math import radians
 import re
 import pytz
+from datetime import datetime, timedelta
+
 
 def _to_string(x):
     x = re.sub('[^A-Za-z0-9,.]', '', str(x))
     return '(' + x + ')'
 
 
+def _sqlite_get_date(connection, installation, function):
```

Comment:
This query might be a bit costly, perhaps it is better to simply remove the date constraints from the data fetching query if no dates are provided?